### ISSUE TITLE
Replays

### ISSUE THREAD

**Issue by denkstrum-hots** [chairleague/hots-rules#17]
I would like to explore the possibility of having a replay repository and requiring replay uploads post match.  It works well for verifying match integrity, but also allows casters to practice games site unseen and scouting.  

Since money will be collected, could a drop box type system with labeled replays be able to be set up?  Maybe a poll of the teams if they are even interested in this feature?

**Comment by johnafogarty4** [comment]
This, I love this.  I've been working on a system for casters that will parse replays and tell you what teams like to draft.  My very, very rough, very manual version of this that I used in the Div 1 finals can be seen here.  https://docs.google.com/spreadsheets/d/1C2vObHwOM5mhnWWftwd7BAo8KGwD3QMP70MSJrSJb64/edit?usp=sharing

When Superjova tweeted about the possibility of an API the wheels started turning.  I'd like to combine my spreadsheet with a masterleague.net type system, so when you go into casting a game, you can have a web page that shows you information similar to what I have in my spreadsheet, but then also including a smaller version of each teams last drafts like ML does.  There is a lot of work to be done for it, but it would be interesting to see if there is any desire for it from casters, and how we integrate with CL system so you can't use it to cheat.

**Comment by superjova** [comment]
Are we proposing uploading of videos or replay files?

**Comment by denkstrum-hots** [comment]
Replay files, it's a universal standard and under a mb in size for each game.

**Comment by Durgulach** [comment]
Having a tool that shows what a team is likely to draft would be an epic scouting tool in addition to being a caster tool.  Doubly so since custom games are not on HotsLogs

**Comment by johnafogarty4** [comment]
I only planned to give that to the casters, and not the players or other teams.

**Comment by Durgulach** [comment]
With the way the site is set up currently though anyone can get a caster account

**Comment by superjova** [comment]
Anyone should be able to upload a replay, but I don't think it should be a requirement.

Edit: anyone from the match. Caster or either team

**Comment by johnafogarty4** [comment]
@Durgulach yes but you can add it to the /caster part of CL site, so that you can only see the data if you're casting the match.

**Comment by Zedalve** [comment]
This would be helpful for adding a stat leaderboard to chairleague
